src/components/EventCardsGridContainer/index.js

```diff
@@ -71,9 +72,11 @@ export default class EventCardsGrid extends Component {
     walletAddresses: PropTypes.array.isRequired,
     intl: intlShape.isRequired, // eslint-disable-line react/no-typos
     status: PropTypes.string,
+    txReturn: PropTypes.object,
   };
 
   static defaultProps = {
+    txReturn: undefined,
```

Comment:
this should be unnecessary since the propType is not `.isRequired`

Reply:
@dwalintukan make txReturn required will throw warning ```Warning: Failed prop type: The prop `txReturn` is marked as required in `EventCardsGrid`, but its value is `undefined`.```

Reply:
oh i see because it is not defined in the redux store first